runtime/balrt.h

```diff
@@ -69,6 +69,10 @@ typedef uint64_t PackedPanic;
 
 typedef uintptr_t PC;
 
+typedef struct {
```

Comment:
Given we are just having a tagged representation, we don't need this any more.

Reply:
Removed.